src/guide/code-owners.md

```diff
@@ -27,182 +26,189 @@ This list is intended to make it easier to identify which email list to include
+  * [man](https://github.com/openjdk/jdk/tree/master/src/java.base/share/man)
+    * [java.1](https://github.com/openjdk/jdk/tree/master/src/src/java.base/share/man/java.1) - Tools, HotSpot
+    * [keytool.1](https://github.com/openjdk/jdk/tree/master/src/src/java.base/share/man/keytool.1) - Security
+  * [[aix](https://github.com/openjdk/jdk/tree/master/java.base/aix/native), [linux](https://github.com/openjdk/jdk/tree/master/java.base/linux/native), [macosx](https://github.com/openjdk/jdk/tree/master/java.base/macosx/native), [share](https://github.com/openjdk/jdk/tree/master/java.base/share/native), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/native), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/native)]/native
+    * [common](https://github.com/openjdk/jdk/tree/master/src/java.base/windows/native/common)
+    * [[share](https://github.com/openjdk/jdk/tree/master/java.base/share/native/include), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/native/include), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/native/include)]/include - Runtime, Core Libs
+    * [jspawnhelper](https://github.com/openjdk/jdk/tree/master/src/java.base/unix/native/jspawnhelper) - Tools
+    * [[share](https://github.com/openjdk/jdk/tree/master/java.base/share/native/launcher), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/native/launcher), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/native/launcher)]/launcher - Tools
+    * [[aix](https://github.com/openjdk/jdk/tree/master/java.base/aix/native/libjava), [linux](https://github.com/openjdk/jdk/tree/master/java.base/linux/native/libjava), [macosx](https://github.com/openjdk/jdk/tree/master/java.base/macosx/native/libjava), [share](https://github.com/openjdk/jdk/tree/master/java.base/share/native/libjava), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/native/libjava), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/native/libjava)]/libjava - Core Libs
+    * [[share](https://github.com/openjdk/jdk/tree/master/java.base/share/native/libjimage), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/native/libjimage), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/native/libjimage)]/libjimage - Core Libs
+    * [[aix](https://github.com/openjdk/jdk/tree/master/java.base/aix/native/libjli), [macosx](https://github.com/openjdk/jdk/tree/master/java.base/macosx/native/libjli), [share](https://github.com/openjdk/jdk/tree/master/java.base/share/native/libjli), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/native/libjli), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/native/libjli)]/libjli - Tools
```

Comment:
"jli" = Java launcher infrastructure so should have same owner as java launcher.

Reply:
Fixed.